tests/marker_test.go

```diff
@@ -0,0 +1,69 @@
+// Copyright 2018 Red Hat, Inc.
```

Comment:
please update year in all files you modified

Reply:
Done